src/configure/helper/devOps/azureDevOpsHelper.ts

```diff
@@ -83,6 +83,8 @@ export class AzureDevOpsHelper {
             };
         }
 
+        let properties = { 'source': 'VSCode' };
```

Comment:
Do you want to specify our extension name here ?

Reply:
Using the complete Id now, as extension name is too generic